zebra-network/src/meta_addr.rs

```diff
@@ -521,6 +521,24 @@ impl MetaAddr {
         is_node && self.address_is_valid_for_outbound()
     }
 
+    /// Should this peer considered unreachable?
+    ///
+    /// A peer is considered unreachable if the last connection attempt to it failed and the last
+    /// successful connection is more than 3 days ago.
+    pub fn is_probably_unreachable(&self) -> bool {
+        self.last_connection_state == PeerAddrState::Failed && self.was_not_recently_seen()
+    }
+
+    /// Was this peer last seen a long time ago?
+    ///
+    /// A long time in this case is more than 3 days ago or never seen before.
+    pub fn was_not_recently_seen(&self) -> bool {
```

Comment:
Naming: Negatives are harder to read and understand.

We'd need to invert the logic in `is_probably_unreachable` as well.

```rust
    pub fn last_seen_is_recent(&self) -> bool {
```

Reply:
Done in [c27243a](https://github.com/ZcashFoundation/zebra/pull/3030/commits/c27243ab655028d2fd325bf70970291908140742).